### PR TITLE
CON-1803: Deprecate non-C2C methods in Workday app

### PR DESCRIPTION
**Pull request by milan-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
@milan-sumo, your updates look good.

**Comment by jpipkin1** [comment]
@kimsauce, could you take a look at this PR? The deploy-to-review check failed, and I'm not sure why.

File: docs/integrations/saas-cloud/workday.md [hunk 1]
```diff
@@ -275,11 +275,11 @@ In this section, we will configure a collection of login and audit logs  from Wo
 * For non-FedRamp Sumo Logic deployments
     * [Create a Workday Cloud To Cloud Collector Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/workday-source).
 * If you're using the Sumo Logic FedRamp deployment, you can either use a Sumo Logic Workday collection script to be run in an Amazon Web Services (AWS) environment using the AWS Lambda service, or run Sumo Logic Python scripts to run periodically on a Linux machine via a cron job.
-    * [AWS Lambda based collection](#Deploy_the_Sumo_Logic_Workday_SAM_Application) via a Serverless Application Model (SAM) application.
-    * [Script-based collection](#Configure_script-based_collection_for_Workday) from a Linux machine.
+    * [AWS Lambda based collection](#deploy-the-sumo-logic-workday-sam-application-in-your-aws-environment-deprecated) via a Serverless Application Model (SAM) application.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/workday.md:278]:
```suggestion
    * [AWS Lambda-based collection](#deploy-the-sumo-logic-workday-sam-application-in-your-aws-environment-deprecated) via a Serverless Application Model (SAM) application.
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi @milan-sumo , I've made some additional style/cleanup changes. Can you please take a look?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**